app/src/main/res/layout/fragment_client_settings.xml

```diff
@@ -1,256 +1,293 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:clickable="true"
-    android:focusable="true"
-    android:focusableInTouchMode="true"
-    android:background="@color/primary_dark">
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/primary_dark"
+        android:clickable="true"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
+        >
 
     <androidx.appcompat.widget.AppCompatTextView
-        android:id="@+id/clientSettingsTitleTv"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:fontFamily="@font/rubik_medium"
-        android:text="@string/settings"
-        android:textColor="@color/white"
-        app:autoSizeTextType="uniform"
-        app:layout_constraintBottom_toTopOf="@+id/clientSettingsTitleTvGLBottom"
-        app:layout_constraintEnd_toStartOf="@+id/closeIbtn"
-        app:layout_constraintStart_toStartOf="@+id/clientSettingsGLBegin"
-        app:layout_constraintTop_toBottomOf="@+id/clientSettingsTitleTvGLTop" />
+            android:id="@+id/clientSettingsTitleTv"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:fontFamily="@font/rubik_medium"
+            android:text="@string/settings"
+            android:textColor="@color/white"
+            app:autoSizeTextType="uniform"
+            app:layout_constraintBottom_toTopOf="@+id/clientSettingsTitleTvGLBottom"
+            app:layout_constraintEnd_toStartOf="@+id/closeIbtn"
+            app:layout_constraintStart_toStartOf="@+id/clientSettingsGLBegin"
+            app:layout_constraintTop_toBottomOf="@+id/clientSettingsTitleTvGLTop"
+            />
 
     <ImageView
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/5d82c3c03be08be1fc2ef44c85e67b13e3e92a8d/app/src/main/res/layout/fragment_client_settings.xml#L28"></span>Missing <code>contentDescription</code> attribute on image</td>
    </tr>
  </tbody>
</table>

Reply:
@gabrysgab maybe you can add tools:ignore to the ImageViews where you don't need contentDescription or if you won't need it anywhere - some exception to Detect

Reply:
I'll leave it for now. But it's a good point to do something with it. I guess it should be a part of discussion about accessibility in app.